.github/workflows/ci.yml

```diff
@@ -5,7 +5,15 @@ on:
     branches:
       - master
   pull_request:
-
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tag to run the workflow on'
+        required: true
+      run_specific_job:
+        description: 'Specify a job to run (test-runtime, try-runtime-execute, etc.)'
+        required: true
```

Comment:
I do not think this is necessary

Reply:
You mean the whole workflow_dispatch?

Reply:
I would just have forked the tag and created a dummy PR, but if this works it is OK for me.